logstash-core/lib/logstash/patches/clamp.rb

```diff
@@ -0,0 +1,47 @@
+require 'clamp'
+require 'logstash/environment'
+
+module Clamp
+  module Attribute
+    class Instance
+      def default_from_environment
+        # we don't want uncontrolled var injection from the environment
```

Comment:
good you added a few context comment 👍 , on the other side to make this even better,  the reason might obvious, but why don't we want it?